src/zodbbrowser/templates/broken.pt

```diff
@@ -0,0 +1,57 @@
+<metal:block metal:use-macro="view/@@zodbbrowser_macros/page">
+  <metal:block fill-slot="content">
+
+    <div class="object"
+         tal:define="available view/isFeatureAvailable">
+
+      <div class="heading">
+        <h1><a href="@@zodbbrowser">Broken ZODB objects</a></h1>
+      </div>
+
+      <tal:documentation tal:condition="not: available">
+        <div>
+          <p>
+            This feature is not available. In order to have it here,
+            you need to first run the zodbcheck script, saving the
+            information generated by this script with the help of the
+            <tt>--save-references</tt> option and give it to
+            zodbbrowser with the option <tt>--load-references</tt>.
+          </p>
+        </div>
+      </tal:documentation>
```

Comment:
Yes, that is what we tried at first. However on our largest database, it does take one hour to build the index. So we wanted to do only one time, save it, and be able to open it and use it directly.

Additionally the zodbcheck script can tell you advance if you have broken objects or not.